packages/orbit-components/src/InputGroup/index.tsx

```diff
@@ -222,8 +222,8 @@ const InputGroup = React.forwardRef<HTMLDivElement, Props>(
     const labelRef = React.useRef(null);
     const iconRef = React.useRef(null);
 
-    const foundErrors = React.useMemo(() => findPropInChild("error", children), [children]);
-    const foundHelp = React.useMemo(() => findPropInChild("help", children), [children]);
+    const foundErrors = findPropInChild("error", children);
+    const foundHelp = findPropInChild("help", children);
```

Comment:
This is one of the cases it can probably make sense to keep the useMemo, because we are iterating over children twice per call. However, the number of children should not be too big on each iteration, so probably it's still worth it…

Did you try any metrics to see which perform better?

Reply:
I run locally profiler, no difference 🤔